perf/ring.go

```diff
@@ -65,6 +60,22 @@ func newPerfEventRing(cpu, perCPUBuffer, watermark int) (*perfEventRing, error)
 	return ring, nil
 }
 
+// mmapBufferSize returns a valid mmap buffer size for use with perf_event_open (1+2^n pages)
+func mmapBufferSize(perCPUBuffer int) int {
```

Comment:
Nit:

```suggestion
func perfBufferPages(perCPUBuffer int) int {
```

Reply:
I changed it to `perfBufferSize()`, is that ok? I thought `perfBufferPages()` sounded like it returned a number of pages.

Reply:
Ack